frontend/src/components/inputs/linkedinloginbutton/LinkedInLoginButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from "@mantine/core";
+import { useLinkedIn } from "react-linkedin-login-oauth2";
+
+const client_id = "78c9i5swfp7ii3";
+const scope = "openid email";
+const redirect_uri = "http://127.0.0.1:3000/auth/callback";
```

Comment:
will this work when deployed? remember, the URL is different on the prod server

Reply:
probably not, something I need to fix for sure.